docs/content/preview/migrate/troubleshoot.md

```diff
@@ -223,6 +223,26 @@ Suggested workaround is as follows:
 
 ---
 
+### Issue #137: [MYSQL] Spatial datatype migration is not yet supported
+
+**GitHub link**: [Issue #137](https://github.com/yugabyte/yb-voyager/issues/137)
+
+**Description**: If your MYSQL schema contains spatial datatypes, the migration will not complete as this migration type is not yet supported by YugabyteDB Voyager. This will require extra dependencies such as [PostGIS](https://postgis.net/) to be installed.
```

Comment:
This will require extra dependencies - what is 'this'? installed why and where? in the future, for a migration that won't work anyway?